draft-ietf-quic-transport.md

```diff
@@ -2254,12 +2254,8 @@ abandons its attempt to validate the path.
 Endpoints SHOULD abandon path validation based on a timer. When setting this
 timer, implementations are cautioned that the new path could have a longer
 round-trip time than the original.  A value of three times the larger of the
-current Probe Timeout (PTO) or the initial timeout (that is, 2*kInitialRtt) as
-defined in {{QUIC-RECOVERY}} is RECOMMENDED.  That is:
-
-~~~
-   validation_timeout = max(3*PTO, 6*kInitialRtt)
-~~~
+current Probe Timeout (PTO) or the PTO for the new path (that is, using
+kInitialRtt as defined in {{QUIC-RECOVERY}}) is RECOMMENDED.
```

Comment:
Can we add the computation back in? With the corrected values of course:

That is, `validation timeout = max(3*PTO, 9*kInitialRtt)`

Reply:
But that is wrong.  Because the PTO on the new path is `9*kInitialRtt` on the handshake, but `9*kInitialRtt + 3*max_ack_delay` on migration.

Reply:
Oh hm. It's actually more than that, since rttvar is also set to kInitialRtt/2 initially. Ok, let's leave it as you have it then.